scaffold/github/workflows/ComposerLockDiffComment.yml

```diff
@@ -0,0 +1,12 @@
+name: "Post Comment on a Github issue/PR with the composer-lock-diff contents"
```

Comment:
I suspect this workflow is just for PRs.

Maybe change the name from

Post Comment on a Github issue/PR with the composer-lock-diff contents

to

Post Comment on a Github PR with the composer-lock-diff contents

Reply:
I think since composer-lock-diff is a command, changing `contents` to `output` might also be a bit more accurate.

Reply:
I looked to see where that string for the name would show up and it seems like here:
<img width="1813" alt="Screen Shot 2022-09-26 at 5 26 47 PM" src="https://user-images.githubusercontent.com/1970586/192383924-74b945f1-9462-4db3-9928-d2092de83e30.png">

Maybe since this is a Github action, and we would see the label inside Github, we can shorten this label, and move the composer-lock-diff to the front, so it shows when trimmed...

Like maybe "Post composer-lock-diff output as a PR comment"

Reply:
I agree.
I had "PR/Issue" because these are treated the same for the GitHub API, and at some point, I thought I was going to pass the `composer-lock-diff` output to the action of posting a comment.